script/makeraw_unix.go

```diff
@@ -0,0 +1,33 @@
+//go:build darwin || dragonfly || freebsd || netbsd || openbsd || aix || linux || solaris || zos
```

Comment:
Shouldn't we add the license identifier at the start of the file? Same for the others new files

Reply:
added.